src/SourceBuild/content/repo-projects/runtime.proj

```diff
@@ -16,7 +16,7 @@
     <RuntimeOS>$(NETCoreSdkRuntimeIdentifier.Substring(0, $(_platformIndex)))</RuntimeOS>
 
     <_platformIndex>$(NETCoreSdkPortableRuntimeIdentifier.LastIndexOf('-'))</_platformIndex>
-    <BaseOS>$(NETCoreSdkPortableRuntimeIdentifier.Substring(0, $(_platformIndex)))</BaseOS>
+    <BaseOS>$(NETCoreSdkPortableRuntimeIdentifier.Substring(0, $(_platformIndex)))-$(Platform)</BaseOS>
```

Comment:
@tmds I had to add the `Platform` property here so that [AdditionalRuntimeIdentifierParent](https://github.com/dotnet/runtime/blob/c643fedf020bfdcd53afee5d4debb57ded5419a6/eng/SourceBuild.props#L45C44-L45C77) is set to the right value.

This needs a change as we now use a different "add current RID to RID graph" code in runtime (the one that originated in sdk): https://github.com/dotnet/runtime/blob/main/src/libraries/Microsoft.NETCore.Platforms/src/UpdateRuntimeIdentifierGraph.cs

Please double check if this is correct. Should we change the name of this property to `PortableRid` (which is what sdk.proj uses)? Changing the property name would require a change in runtime.

Reply:
> This needs to change as we now use a different "add current RID to RID graph" code in runtime (the one that originated in sdk): https://github.com/dotnet/runtime/blob/main/src/libraries/Microsoft.NETCore.Platforms/src/UpdateRuntimeIdentifierGraph.cs

Yes, it seems that the task that updated the old runtime graph didn't expect this to include an architecture (which is why this is named `OS` and not `Rid`), while the new task that updates the new graphs expects this to include an architecture.

> Please double check if this is correct. Should we change the name of this property to PortableRid (which is what sdk.proj uses)? Changing the property name would require a change in runtime.

If it includes an architecture, it makes more sense to end with a `Rid` suffix.

Yes, this line needs to change: https://github.com/dotnet/runtime/blob/487d7f010b90a82178d55eea21e72dc6dedf9d5d/eng/SourceBuild.props#L45.